UnityProject/Assets/Scripts/Items/Others/Restraint.cs

```diff
@@ -68,7 +68,8 @@ void ProgressFinishAction()
 			.ServerStartProgress(target.RegisterTile(), applyTime, performer);
 		if (bar != null)
 		{
-			SoundManager.PlayNetworkedAtPos(sound, target.transform.position, sourceObj: target.gameObject);
+			// JESTER
+			//SoundManager.PlayNetworkedAtPos(sound, target.transform.position, sourceObj: target.gameObject);
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/gh/unitystation/unitystation/pullRequest?prid=6280689)